testsuite/src/main/java/io/netty/testsuite/transport/AbstractComboTestsuiteTest.java

```diff
@@ -21,37 +21,29 @@
 import io.netty.util.internal.StringUtil;
 import io.netty.util.internal.logging.InternalLogger;
 import io.netty.util.internal.logging.InternalLoggerFactory;
-import org.junit.Rule;
-import org.junit.rules.TestName;
+import org.junit.jupiter.api.TestInfo;
 
-import java.lang.reflect.InvocationTargetException;
-import java.lang.reflect.Method;
 import java.util.List;
 
 public abstract class AbstractComboTestsuiteTest<SB extends AbstractBootstrap<?, ?, ?>,
         CB extends AbstractBootstrap<?, ?, ?>> {
-    private final Class<SB> sbClazz;
-    private final Class<CB> cbClazz;
     protected final InternalLogger logger = InternalLoggerFactory.getInstance(getClass());
     protected volatile CB cb;
     protected volatile SB sb;
 
-    protected AbstractComboTestsuiteTest(Class<SB> sbClazz, Class<CB> cbClazz) {
-        this.sbClazz = sbClazz;
-        this.cbClazz = cbClazz;
-    }
-
     protected abstract List<TestsuitePermutation.BootstrapComboFactory<SB, CB>> newFactories();
 
     protected List<ByteBufAllocator> newAllocators() {
         return TestsuitePermutation.allocator();
     }
 
-    @Rule
-    public final TestName testName = new TestName();
-
     protected void run() throws Throwable {
+        throw new UnsupportedOperationException();
+    }
```

Comment:
This method is still called by Epoll and KQueue modules - it will be removed once those modules are migrated.